runtime/src/inline_feature_gate_program.rs

```diff
@@ -0,0 +1,5 @@
+//! Contains replacement program IDs for the feature gate program
+
+pub(crate) mod noop_program {
+    solana_sdk::declare_id!("2rqZsQBbacRbuAuTSuJ7n49UQT9fzes8RLggFcmB9YuN");
```

Comment:
As a part of this, should we start including the new feature-gate program in:
1. https://github.com/solana-labs/solana/blob/master/fetch-spl.sh
2. https://github.com/solana-labs/solana/blob/master/program-test/src/programs.rs
for new clusters?

Reply:
Yeah, probably! Since it's not a native program. 

With that being said, is it weird that we'd have cluster operations dependent on a non-native program?

Reply:
In this case I think it's ok. But by adding the program elf in those two places, the program should be added to new genesis configs that are created when a test cluster is booted, so the program will always be present practically

Reply:
I think in order to add the program here, we'd have to merge [this PR](https://github.com/solana-labs/solana-program-library/pull/4987) and then publish a release for the Feature Gate program, right?